src/bindings/io/README.md

```diff
@@ -0,0 +1,12 @@
+# I/O Bindings
```

Comment:
What do you think about including this information to the README.md one level above?

Having it here would be possible, too. But then the README.md should be longer (to avoid the "too many tabs open" problem we already got complaints about) and somehow needs to be included into the website.